cmd/kn/main.go

```diff
@@ -64,21 +64,26 @@ func run(args []string) error {
 		return err
 	}
 
-	// Strip of all flags to get the non-flag commands only
-	commands, err := stripFlags(args)
+	pluginManager := plugin.NewManager(config.GlobalConfig.PluginsDir(), config.GlobalConfig.LookupPluginsInPath())
+
+	// Create kn root command and all sub-commands
+	rootCmd, err := root.NewRootCommand(pluginManager.HelpTemplateFuncs())
 	if err != nil {
 		return err
 	}
```

Comment:
This error is not covered in tests...

Reply:
thats for creating rootCommand, while we could add (duplicate) test for it here I wonder if its worth as its tested in `pkg/kn/root/root_test.go`.